test/scripts/e2e_client_runner.py

```diff
@@ -447,13 +447,12 @@ def main():
         create_kmd_config_with_unsafe_scrypt(env['ALGORAND_DATA'])
         create_kmd_config_with_unsafe_scrypt(env['ALGORAND_DATA2'])
 
-
     xrun(['goal', '-v'], env=env, timeout=5)
     xrun(['goal', 'node', 'status'], env=env, timeout=5)
 
     rs = RunSet(env)
     for scriptname in args.scripts:
-        rs.start(scriptname, args.timeout-10)
+        rs.start(os.path.abspath(scriptname), args.timeout-10)
```

Comment:
I think that this is a good change, but I'm not sure it has to be coupled with the rest of this PR.